oemof/solph/components.py

```diff
@@ -152,6 +152,19 @@ def __init__(self, *args,
         if self._invest_group is True:
             self._check_invest_attributes()
 
+        # Check for old parameter names.
```

Comment:
I thought about putting this to a central location, e.g. `oemof.network.Node.__init__`. However, as `max_storage_level` and `min_storage_level` are not passed to the parent constructors, this location is optimal. (Maybe, you could add a comment to explain this.)

Reply:
I do not understand why you want to move this to the Node class. For me it is only related to the GenericStorage and only related to the update process from 0.2.x to 0.3.x. Therefore I think it is a temporal code block and we should keep on discussing a general solution.

BTW: I have no idea why `max_storage_level` and `min_storage_level` are optional parameters and not covered by `kwargs` as all the other parameters.

Reply:
Parameters in kwargs: That's inconsistent anyway. See `ExtractionTurbineCHP`, for another example.

Locating check in node: Similar things could happen with other releases. So it might help. Still, a general solution is of course preferable. #560 really should be resolved before the next major release...

Reply:
As this is a temporal hack I would let it where it is and add a comment that it is temporal.